src/common/Graylog.cc

```diff
@@ -24,7 +24,7 @@ Graylog::Graylog(const SubsystemMap * const s, std::string logger)
       m_log_dst_valid(false),
       m_hostname(""),
       m_fsid(""),
-      m_logger(logger),
+      m_logger(std::move(logger)),
```

Comment:
sorry, i was wrong. thought that the type of `logger` should be `std::string&&`.

> but a copy is a copy, and I'd prefer to move when it's appropriate

makes sense to me.